recipes/default.rb

```diff
@@ -24,10 +24,21 @@
 # ::Chef::Recipe.send(:include, Chef::Mixin::PowershellOut)
 ::Chef::Resource::RubyBlock.send(:include, Chef::Mixin::PowershellOut)
 
-powershell_script 'install chocolatey' do
-  code "iex ((new-object net.webclient).DownloadString('#{node['chocolatey']['Uri']}'))"
-  convert_boolean_return true
-  not_if { ChocolateyHelpers.chocolatey_installed? }
+if File.exist?('C:\windows\system32\cmd.exe')
+  arch = :x86_64
+  cmd = 'C:\windows\system32\cmd.exe'
```

Comment:
No it doesn't exist outside of syswow64. I think it will need additional checks for architecture that are more deterministic, but consider the following:

The only time you need to make choco do something different is when syswow64 is at play, the other times it will just work as intended.